src/client/components/ChannelInformation/ChannelInnformation.js

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import './ChannelInformation.style.css';
 
 export const RenderChannelInformation = ({
-  headline = 'Here is channel info',
+  channelImg,
+  channelTitle,
+  members,
 }) => {
-  return <h3>{headline}</h3>;
+  return (
+    <>
+      <div className="channel-information-header">
+        <img src={channelImg} alt="Room" className="channel-information-img" />
+      </div>
+      <h3 className="channel-information-title">{channelTitle}</h3>
+      <div className="channel-information-main">
+        {members &&
+          members.map((member) => {
+            return (
+              <div>
+                <img
+                  src={member.imgUrl}
+                  alt="member"
+                  className="member-information-img"
+                />
+                <h4>{member.name}</h4>
+              </div>
+            );
+          })}
+      </div>
+    </>
+  );
+};
+RenderChannelInformation.propTypes = {
+  channelImg: PropTypes.string,
+  channelTitle: PropTypes.string,
+  members: PropTypes.arrayOf(PropTypes.object),
+};
+RenderChannelInformation.defaultProps = {
+  channelImg: 'https://loremflickr.com/320/240',
+  channelTitle: 'Room',
+  members: [
+    { name: 'Member', imgUrl: 'https://loremflickr.com/320/240' },
```

Comment:
This really isn't a piece of default information. You can and you should use these in storybook instead.
Default props are used to initialize default values such as `""`, `0`, `[]`, `{}` and `false`.